### PR TITLE
Use Axios to send Http request; Update Service address; Add encode field in request body

### PR DESCRIPTION
**Pull request by Angus-F**

### CODE REVIEW

File: src/components/Shorten.js [hunk 1]
```diff
@@ -19,8 +28,8 @@ const Shorten = () => {
 
   const isAnlyUrl = (url) => {
     if (
-      url.includes("http://localhost:8080/anly") ||
-      url.includes("https://localhost:8080/anly/")
+      url.includes("http://localhost:8080") ||
```
Comment (LeonQC) [inline src/components/Shorten.js:31]: Put this into a Constant.js

File: src/components/Shorten.js [hunk 2]
```diff
@@ -19,8 +28,8 @@ const Shorten = () => {
...
+      url.includes("https://localhost:8080")
```
Comment (LeonQC) [inline src/components/Shorten.js:32]: Same as above

**Comment by LeonQC** [comment]
Test evidence please

**Comment by Angus-F** [comment]
> Test evidence please

4441681088254_ pic
4451681088264_ pic
4481681088547_ pic

**Review by LeonQC — APPROVED** [review]

**Pull request merged**